server/templates.go

```diff
@@ -175,6 +179,14 @@ func (t *templates) approval(w http.ResponseWriter, state, username, clientName
 	renderTemplate(w, t.approvalTmpl, data)
 }
 
+func (t *templates) oob(w http.ResponseWriter, code string) {
```

Comment:
@dghubble I still need to move the "clients" document over https://github.com/coreos/dex/blob/master/Documentation/clients.md